### ISSUE TITLE
Module re-arrangement to follow PEP import guideline

### ISSUE THREAD

**Issue by yijiangh** [yijiangh/pybullet_planning#4]
No import inside functions...

**Comment by caelan** [comment]
Do you think it's beneficial to strictly adhere to PEP style at this point in time? For projects with few developers, I typically selectively use PEP elements out of convenience.

**Comment by yijiangh** [comment]
@caelan Agree. I am not planning to do this now but just leave a reminder for myself here.
